### PR TITLE
Add support page

### PR DESCRIPTION
**Pull request by eternoendless**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description  | Add a page to help people find support resources

Screenshot:

Screenshot

### CODE REVIEW

File: content/support/_index.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+* [Report a bug on GitHub]({{< devdocs "contribute/contribute-reporting-issues/" >}})
+* [Join the community forums](https://www.prestashop.com/forums/)
+* [Ask on Stack Overflow](https://stackoverflow.com/questions/tagged/prestashop)
+* [Discuss on Slack]({{< relref "/slack" >}})
+
+## Contact a professional
+
+If you need serious help with your project, your best choice is to get contact a professional. There are thousands of agencies and freelance experts located around the globe:
+
+* [PrestaShop partner agencies and experts](https://www.prestashop.com/experts)
```
Comment (Progi1984) [inline content/support/_index.md:24]:
```suggestion
```

File: content/support/_index.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+* [Find a community expert on Slack]({{< relref "/slack" >}}) (within the `#jobs` channel)
+
+## Learn PrestaShop
+
+"Give a man a fish and you feed him for a day; teach a man to fish and you feed him for a lifetime."
+
+* [Training courses](https://www.prestashop.com/training)
```
Comment (Progi1984) [inline content/support/_index.md:31]:
```suggestion
```

**Review by Progi1984 — CHANGES_REQUESTED** [review]
Two small comments.

I remove all links to prestashop.com in waiting for how PrestaShop.com and all foundations members will be gain to invest in project. May be @ow2 can answer to that.

File: content/support/_index.md (hunk 1 above)
Comment (matks) [reply content/support/_index.md:24]: I disagree with removing the link, but I think the link should be clear whether it's PrestaShop company or PrestaShop project.

```suggestion
* [PrestaShop SA partner agencies and experts](https://www.prestashop.com/experts)
```

File: content/support/_index.md (hunk 2 above)
Comment (matks) [reply content/support/_index.md:31]: Similarly to what I said previously
```suggestion
* [PrestaShop SA training courses](https://www.prestashop.com/training)
```

File: content/support/_index.md (hunk 1 above)
Comment (eternoendless) [reply content/support/_index.md:24]: How about this?
```suggestion
* [List of agencies and experts](https://www.prestashop.com/experts) (sponsored by PrestaShop SA)
```

File: content/support/_index.md (hunk 2 above)
Comment (matks) [reply content/support/_index.md:31]: Better 👍

File: content/support/_index.md (hunk 1 above)
Comment (matks) [reply content/support/_index.md:24]: Better 👍

File: content/support/_index.md (hunk 2 above)
Comment (okom3pom) [inline content/support/_index.md:31]: 404 error you need to add `/en/` but i also agree @Progi1984

File: content/support/_index.md (hunk 1 above)
Comment (okom3pom) [reply content/support/_index.md:24]: 404 error you need to add `/en/` but i also agree @Progi1984

**Review by okom3pom — APPROVED** [review]

**Comment by matks** [comment]
Drupal does this and it's fine
https://www.drupal.org/drupal-services
https://www.drupal.org/docs/troubleshooting-drupal/how-to-ask-for-support-from-the-drupal-community

**Review by Progi1984 — APPROVED** [review]

File: content/support/_index.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+PrestaShop has a sizeable community, and there are many volunteers willing to help each other on the web:
+
+* [Report a bug on GitHub]({{< devdocs "contribute/contribute-reporting-issues/" >}})
+* [Join the community forums](https://www.prestashop.com/forums/)
+* [Ask on Stack Overflow](https://stackoverflow.com/questions/tagged/prestashop)
+* [Discuss on Slack]({{< relref "/slack" >}})
+
+## Contact a professional
+
+If you need serious help with your project, your best choice is to get contact a professional. There are thousands of agencies and freelance experts located around the globe:
```
Comment (matks) [inline content/support/_index.md:22]: "your best choice is to get contact a professional"

I think maybe there's a little typo here?

-> your best choice is to get in touch with a professional
-> your best choice is to contact a professional

**Review by matks — APPROVED** [review]
Minor issue but it's mergeable IMO

File: content/support/_index.md (hunk 3 above)
Comment (eternoendless) [reply content/support/_index.md:22]: Good catch, thanks
```suggestion
If you need serious help with your project, your best choice is to contact a professional. There are thousands of agencies and freelance experts located around the globe:
```

File: content/support/_index.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
+---
+title: "Support"
+---
+
+# I need help with PrestaShop
```
Comment (eternoendless) [inline content/support/_index.md:5]:
```suggestion
# Get help with PrestaShop
```

**Review by matks — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Pull request merged**